usecases/scheduler/src/lambda/ssv2_configUpdater/configUpdater.py

```diff
@@ -0,0 +1,274 @@
+import boto3, botocore
+from boto3.dynamodb.types import TypeDeserializer as ddbDeserializer
+import os
+import json
+from datetime import datetime
+
+## Environment Variables
+## Initialize
+region = os.environ['SSV2_REGION']
+s3Bucket = os.environ['SSV2_S3_BUCKET']
+snsArnPrefix = os.environ['SSV2_SNSARN_PREFIX']
+ebrolesARN = os.environ['SSV2_EVENTBRIDGE_ROLES_ARN']
+jobDef = os.environ['SSV2_JOB_DEF']
+jobQueue = os.environ['SSV2_JOB_QUEUE']
+schedulerGroupName = os.environ['SSV2_SCHEDULER_NAME']
+
+s3 = boto3.client('s3', region_name=region)
+sns = boto3.client('sns', region_name=region)
+scheduler = boto3.client('scheduler', region_name=region)
+
+successResp = {
+    'statusCode': 200,
+    'body': 'success'
+}
+
+##  
+def lambda_handler(event, context):
+    # If event is Insert item to DDB
+    if event['Records'][0]['eventName'] in ('INSERT', 'MODIFY'):
+        items = sanitizeEvent(event, 'INSERT')
+        for item in items:
+            configId = item['configId']
+            emails = item['emails']
+            ssparams = item['ssparams']
+            cronPattern = item['frequency']
+            crossAccounts = item['crossAccounts']
+
+            print('Patching the following Config: {}'.format(configId))
+            result = updateSnsRecipient(configId, emails)
+            if result == False:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)

Similar issue at line numbers 46, 59, and 65.